relaynet-jvm/src/main/java/tech/relaycorp/relaynet/CargoDeliveryRequest.kt

```diff
@@ -0,0 +1,5 @@
+package tech.relaycorp.relaynet
```

Comment:
I noticed you upgraded the library above -- Why do we need to copy this class here?

Reply:
Couldn't import it into the `relaynet-cogrpc-jvm` project, only in the `app` project. So I duplicated the classe until we either:
- Take the project `relaynet-cogrpc-jvm` out of this repo
- Fix the `relaynet-jvm` java version conflict with this project

Reply:
Gotcha.

We have to do both anyway, but the latter is more urgent so I'll focus on that for now. I've just managed to reproduce it: https://github.com/relaycorp/relaynet-cogrpc-jvm/pull/14

Let's keep this class here for the time being.

Reply:
This is now fixed in `tech.relaycorp:relaynet:1.12.4`. Can you please try it so we can remove the code duplication?